src/System.Collections.Immutable/src/Properties/SecurityTransparent.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Security;
+
+[assembly: SecurityTransparent]
```

Comment:
@weshaggard, @joshfree, are there any other corefx assemblies we should be doing this with?  I'm wondering if we should move this file to src/Common.